make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -191,6 +191,9 @@ ifeq ($(call isTargetOs, windows macosx), false)
 
     LIBAWT_XAWT_EXCLUDES := medialib debug
 
+    BUILD_LIBPIPEWIRE_HEADER_DIRS := \
```

Comment:
This variable name indicates that LIBPIPEWIRE is a library that we would build, but it seems to just be headers. I would suggest dropping the `BUILD_` prefix.

Reply:
Updated.